sound/soc/codecs/hdac_hdmi.c

```diff
@@ -119,7 +119,8 @@ struct hdac_hdmi_dai_port_map {
  * pin to port mapping table where the value indicate the pin number and
  * the index indicate the port number with 1 base.
  */
-static const int icl_pin2port_map[] = {0x4, 0x6, 0x8, 0xa, 0xb};
```

Comment:
I dont quite understand this change but the comment above needs to be fixed if the change is valid.

Reply:
and oh there are 2 typos in the commit message "functiton and pin2port mapping talbe"

Reply:
yes, this is the part that leaves me scratching my head. What was the intent and does this still work on ICL?

Reply:
> yes, this is the part that leaves me scratching my head. What was the intent and does this still work on ICL?

I believe that it still work on ICL since @xiulipan also change the way to map pin and port, too. However, I don't get why we need the change?

Reply:
@bardliao pardon my ignorance but what is the pin2port map for? Why do we not have one for GLK or CML and only for ICL and TGL?